app/assets/javascripts/discourse/tests/acceptance/user-preferences-interface-test.js

```diff
@@ -147,6 +148,14 @@ acceptance(
           success: "OK",
         });
       });
+      server.get("/color-scheme-stylesheet/3.json", () => {
+        return helper.response({
+          new_href: "3.css",
+        });
+      });
```

Comment:
Just out of curiosity, why do we need this when we do not expect the client to load the stylesheet?

Reply:
Yes, we don't need it currently. We would need this only if the test goes wrong.

Reply:
In this case, I'll definitely recommend adding a positive test case which assert that the theme preview is loaded.

Reply:
Sure, it's done. https://github.com/discourse/discourse/pull/17855/commits/77f04514a1dd6185008e02b7a86780420c74f822